etna/ciim/client.py

```diff
@@ -134,6 +152,64 @@ def prepare_filter_aggregations(items: Optional[list]) -> Optional[str]:
     return filter_prepared_list
 
 
+class ResultList:
+    """
+    Wrapper class for JSON-decoded list of search results, that lazily converts
```

Comment:
re : list of _search results_ 
Its covering results from various api's for `fetch` and `search`
Suggestion: Wrapper class for JSON-decoded list of _various api search, fetch results_, that lazily converts